plugin/src/main/java/com/iridium/iridiumskyblock/Persist.java

```diff
@@ -40,7 +40,7 @@ public Persist(PersistType persistType, JavaPlugin javaPlugin) {
      * @return The name of the configuration file
      */
     private static String getName(Class<?> clazz) {
-        return clazz.getSimpleName().toLowerCase();
+        return clazz.getSimpleName().toLowerCase().replaceAll("ı", "i");
```

Comment:
Whats this?

Reply:
![image](https://user-images.githubusercontent.com/69310169/118165613-260f5880-b42d-11eb-86df-f3db218a40ab.png)
![image](https://user-images.githubusercontent.com/69310169/118165621-29a2df80-b42d-11eb-80ec-70be6e814713.png)
for file names

Reply:
its converting "I" to "ı"

Reply:
but why?

Reply:
may its about my pc's Locale, i've change it to Locale.English